pipelines/deployer/deployer.yaml

```diff
@@ -462,6 +462,10 @@ drain_cluster_task: &drain_cluster_task
       echo "fetching kubeconfig from aws..."
       aws eks update-kubeconfig --name "${CLUSTER_NAME}" --kubeconfig ./kubeconfig
       export KUBECONFIG=$(pwd)/kubeconfig
+
+      export RESOURCE_TYPES=$(kubectl get crd -o json | jq -r '.items[] | select (.spec.group | endswith(".govsvc.uk")) | .spec.names.kind')
+      kubectl delete -A --all $(echo $RESOURCE_TYPES | tr '\n' ',')
```

Comment:
```suggestion
      kubectl delete -A --all $(echo $RESOURCE_TYPES | tr ' ' ',')
```

Reply:
When I run the command for RESOURCE_TYPES locally (with `gds sandbox`) I get them delimited by newlines, as one would expect from JQ.

Reply:
ah but when I put it into RESOURCE_TYPES itself I get spaces... huh